src/features/CreateGame.js

```diff
@@ -30,14 +29,20 @@ export function CreateGame() {
   );
   const { name, options, customCards } = CreateGameState;
 
+  useEffect(() => {
+    // Note for custom cards selection
+    // This useEffect function will run every time the customCards state changes
+    // This is because the customCards state is added to the dependency array for this useEffect function
+    // Do some conditional logic here and it should be good. And make sure the state is updated onChange of the checkbox
+
+    axios
+      .get("http://3.101.13.217:45000/cards/prompt")
```

Comment:
FYI, you changed the ip to localhost on line 10 of src/UI/Hero.js.  Don't know which one you want or if you want it to be consistent?